k8s/broker/logger_test.go

```diff
@@ -0,0 +1,9 @@
+package broker
+
+import (
+	"github.com/juju/loggo"
+)
+
+func init() {
+	logger.SetLogLevel(loggo.TRACE)
+}
```

Comment:
If we want trace enabled for tests, should we maybe just set that once somewhere (i.e. for the root logger)? Or is that easier said than done because of how Go isolates tests per package?